samples/meeting-av-filters/csharp/README.md

```diff
@@ -0,0 +1,105 @@
+---
+page_type: sample
+description: "This sample illustrates usage av filters for teams meeting."
+products:
+- office-teams
+- office
+- office-365
+languages:
+- csharp
+extensions:
+  contentType: samples
+  createdDate: "01/08/2022 2:36:57 PM"
+urlFragment: officedev-microsoft-teams-samples-meeting-av-filters-csharp
+---
+
+# Meeting AV Filters
+
+This sample demos the use of av filters for teams meetings. 
+
+![filter-page](MeetingAVFilter/Images/MeetingFilterPage.png)
```

Comment:
Let's create a .gif image also and add it.
Also, in sample.json add reference to this .gif.